From 7a86a11017d8237534fa45d5f3b44ca87bdac0db Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@apache.org>
Date: Tue, 6 Dec 2016 13:11:03 -0800
Subject: [PATCH 2098/2848] MAPREDUCE-6571. JobEndNotification info logs are
 missing in AM container syslog (haibochen via
 rkanter)

(cherry picked from commit 6b4a9970f187ab399adf7469e494a530f08993b5)
(cherry picked from commit 1585b4f7cef584c932e2bbd0b8c6451449847fb0)

Change-Id: I4a9ef54366d962ecf9fc796f9878cfef5650aefc
---
 .../hadoop/mapreduce/v2/app/MRAppMaster.java       |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/MRAppMaster.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/MRAppMaster.java
index f15c929..eb73dfc 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/MRAppMaster.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/src/main/java/org/apache/hadoop/mapreduce/v2/app/MRAppMaster.java
@@ -143,7 +143,6 @@
 import org.apache.hadoop.yarn.util.Clock;
 import org.apache.hadoop.yarn.util.ConverterUtils;
 import org.apache.hadoop.yarn.util.SystemClock;
-import org.apache.log4j.LogManager;
 
 import com.google.common.annotations.VisibleForTesting;
 
@@ -1141,11 +1140,10 @@ protected void serviceStart() throws Exception {
       startJobs();
     }
   }
-  
+
   @Override
   public void stop() {
     super.stop();
-    TaskLog.syncLogsShutdown(logSyncer);
   }
 
   private boolean isRecoverySupported() throws IOException {
@@ -1645,7 +1643,6 @@ private void createJobClassLoader(Configuration conf) throws IOException {
   @Override
   protected void serviceStop() throws Exception {
     super.serviceStop();
-    LogManager.shutdown();
   }
 
   public ClientService getClientService() {
-- 
1.7.9.5

